src/frontend/components/SøknadsSteg/Oppsummering/OppsummeringSteg/OmDegOppsummering.tsx

```diff
@@ -59,39 +54,33 @@ const OmDegOppsummering: React.FC<Props> = ({ settFeilAnchors }) => {
                 }
             />
             <OppsummeringFelt
-                tittel={<SpråkTekst id={'felles.fødsels-eller-dnummer.label'} />}
+                tittel={plainTekst(omDegTekster.ident)}
                 søknadsvar={søknad.søker.ident}
             />
             <OppsummeringFelt
-                tittel={<SpråkTekst id={omDegPersonopplysningerSpråkId.søkerStatsborgerskap} />}
+                tittel={plainTekst(omDegTekster.statsborgerskap)}
                 søknadsvar={søknad.søker.statsborgerskap
                     .map((statsborgerskap: { landkode: Alpha3Code }) =>
                         landkodeTilSpråk(statsborgerskap.landkode, valgtLocale)
                     )
                     .join(', ')}
             />
             <OppsummeringFelt
-                tittel={<SpråkTekst id={omDegPersonopplysningerSpråkId.søkerSivilstatus} />}
+                tittel={plainTekst(omDegTekster.sivilstatus)}
                 søknadsvar={søknad.søker.sivilstand.type}
             />
             <OppsummeringFelt
-                tittel={<SpråkTekst id={omDegPersonopplysningerSpråkId.søkerAdresse} />}
+                tittel={plainTekst(omDegTekster.adresse)}
                 children={genererAdresseVisning(søknad.søker, omDegTekster, plainTekst)}
             />
             {søknad.søker.borPåRegistrertAdresse.svar && (
                 <OppsummeringFelt
-                    tittel={
-                        <SpråkTekst
-                            id={omDegSpørsmålSpråkId[OmDegSpørsmålId.borPåRegistrertAdresse]}
-                        />
-                    }
+                    tittel={<TekstBlock block={omDegTekster.borPaaRegistrertAdresse.sporsmal} />}
```

Comment:
Kan disse neste titlene også kun være `plainTekst`, i stedet for `TekstBlock`?

Reply:
`plainTekst` brukes fordi tekstene er `localeRecordString`, altså de har ingen formatering.

Akkurat nå har heller ikke spørsmålene noen formatering, så de vil fungere med `plainTekst` også. Men dersom vi sener endrer spørsmålene i Sanity til å ha formatering, må `TekstBlock` brukes for at de skal vises på riktig måte. Derfor fortsetter vi å ta i bruk `TekstBlock` selv om det ikke utgjør noen forskjell per nå.